src/styles/stars.scss

```diff
@@ -0,0 +1,19 @@
+.card__stars {
```

Comment:
The BEM block should be named 'stars' instead of 'card__stars' as mentioned in the task description.